items.py

```diff
@@ -172,51 +264,46 @@
                    f'--password-file /etc/restic/password_{backup_host} '
                    f'-r {repository_url} '
                    'init',
-        'unless': f'. /etc/restic/env_{backup_host} && /opt/restic/restic -r {repository_url} cat config',
+        'unless': f'set -a; . /etc/restic/env_{backup_host}; set +a && /opt/restic/restic -r {repository_url} cat config',
         'needs': [
             f'download:/opt/restic/restic_{RESTIC_VERSION}.bz2',
             f'tag:prepare_restic_backup_{backup_host}',
             'action:unpack_restic',
             f'file:/etc/restic/password_{backup_host}',
             f'file:/etc/restic/env_{backup_host}',
         ],
+        'tags': {
+            'init_restic'
+        },
     }
 
-    # cron does not like . in filenames
-    files['/etc/cron.hourly/restic_{host_name}'.format(host_name=backup_host.replace('.', '_'))] = {
```

Comment:
please delete the old files.. otherwise we have cron and systemd on migrated systems